scripts/build-validations.ts

```
@@ -0,0 +1,581 @@
+      const srcDir = path.join(this.rootDir, "src");
+      const files = this.getTypeScriptFiles(srcDir, false); // include test directories
+
+      const envVars = new Set<string>();
+
+      for (const file of files) {
+        const fullPath = path.join(this.rootDir, file);
+        if (!fs.existsSync(fullPath)) continue;
+
+        const content = this.readFileWithCache(fullPath);
+        if (!content) continue;
+        let match;
```

Biome (2.3.14)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)